### PR TITLE
(PC-5533) Tracking - Clics sur le 'Bloc Métier'

### PR DESCRIPTION
**Pull request by antoinewg**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-5533

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added a **screenshot** for UI tickets.

## Screenshots

image

### CODE REVIEW

File: src/features/home/components/tests/BusinessModule.test.tsx [hunk 1]
```diff
@@ -54,6 +56,12 @@ describe('BusinessModule component', () => {
   })
 
   it('should open url when clicking on the image', () => {
+    const { getByTestId } = renderModule(props)
+    fireEvent.press(getByTestId('imageBusiness'))
+    expect(logClickBusinessBlock).toHaveBeenCalledWith(props.title)
+  })
+
+  it('should log "BusinessBlockClicked" when clicking on the image', () => {
```
Comment (Giregm) [inline src/features/home/components/tests/BusinessModule.test.tsx]: Il me semble que les deux titres des tests sont inversés

File: src/features/home/pages/useDisplayedHomeModules.test.ts [hunk 2]
```diff
@@ -49,6 +49,7 @@ const notInAlgoliaModules = new Offers({
 const offerModules: ProcessedModule[] = [visibleOfferModule, hiddenOfferModule, emptyHits]
 
 const connectedBusinessModule = new BusinessPane({
+  title: 'mmodule title',
```
Comment (Giregm) [inline src/features/home/pages/useDisplayedHomeModules.test.ts:52]: petite typo

**Review by Giregm — APPROVED** [review]

File: src/features/home/components/tests/BusinessModule.test.tsx (hunk 1 above)
Comment (Giregm) [reply src/features/home/components/tests/BusinessModule.test.tsx]: bon bah t'as rebase entretemps xD

Comment (antoinewg) [reply src/features/home/components/tests/BusinessModule.test.tsx]: j'ai changé juste après :)

**Pull request merged**